src/ray/rpc/raylet/raylet_client.cc

```diff
@@ -358,8 +361,8 @@ ray::Status RayletClient::RegisterClient() {
   return GrpcStatusToRayStatus(status);
 }
 
+// Send heartbeat request to raylet server.
```

Comment:
Comments that describe the method functionality should always go in the header file.